File: templates/Uwp/_comp/MVVMToolkit/Page.AddBackCommand/ViewModels/wts.ItemNameDetailViewModel_postaction.cs
//{[{
using System.Windows.Input;
using Microsoft.Toolkit.Mvvm.ComponentModel;
using Microsoft.Toolkit.Mvvm.Input;
using Param_RootNamespace.Helpers;
using Param_RootNamespace.Services;
//}]}

namespace Param_RootNamespace.ViewModels
{
    public class wts.ItemNameDetailViewModel : ObservableObject
    {
//^^
//{[{
        private ICommand _goBackCommand;

        public ICommand GoBackCommand => _goBackCommand ?? (_goBackCommand = new RelayCommand(OnGoBack));
//}]}

        public wts.ItemNameDetailViewModel()
        {
        }

//^^
//{[{
        private void OnGoBack()
        {
            if (NavigationService.CanGoBack)
            {
                NavigationService.GoBack();
            }
        }
//}]}
    }
}

File: code/TemplateStudioForUWP/Templates/Uwp/_comp/MT/Page.AddBackCommand._VB/ViewModels/wts.ItemNameDetailViewModel_postaction.vb
'{[{
Imports Param_RootNamespace.Services
Imports Microsoft.Toolkit.Mvvm.ComponentModel
Imports Microsoft.Toolkit.Mvvm.Input
'}]}
Namespace ViewModels
    Public Class wts.ItemNameDetailViewModel
        Inherits ObservableObject

        '^^
        '{[{

        Public ReadOnly Property GoBackCommand As ICommand = new RelayCommand(AddressOf OnGoBack)
        '}]}

        Public Sub New()
        End Sub

        '^^
        '{[{
        Private Sub OnGoBack()
            If NavigationService.CanGoBack Then
                NavigationService.GoBack()
            End If
        End Sub
        '}]}
    End Class
End Namespace
